Домашняя работа Урок 2/8/recursiya.js

```diff
@@ -0,0 +1,11 @@
+var val = +prompt('Введите число:');
+var pow = +prompt('Введите степень:');
+
+function power(val, pow) {
+	if(pow === 0) {
+		return 1;
+	}
+	return val**pow;
```

Comment:
Это не python - тут так не работает)
```suggestion
	return val**pow;
```